### ISSUE TITLE
fails to build for 64-bit Windows with rust 1.69

### ISSUE THREAD

**Issue by jeremyd2019** [podsvirov/mdbook-pikchr#1]
Due to https://github.com/MSxDOS/ntapi/issues/15:
```
  error[E0793]: reference to packed field is unaligned
      --> C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\ntapi-0.3.6\src\ntexapi.rs:2785:52
       |
  2785 |         *tick_count.QuadPart_mut() = read_volatile(&(*USER_SHARED_DATA).u.TickCountQuad);
       |                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       |
       = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
       = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)
  
  error[E0793]: reference to packed field is unaligned
      --> C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\ntapi-0.3.6\src\ntexapi.rs:2809:25
       |
  2809 |         ((read_volatile(&(*USER_SHARED_DATA).u.TickCountQuad)
       |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       |
       = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
       = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)
  
  For more information about this error, try `rustc --explain E0793`.
  error: could not compile `ntapi` due to 2 previous errors
```

Maybe it just needs to update the Cargo.lock?

**Comment by podsvirov** [comment]
Thanks for the report! I will update dependencies in my spare time (hopefully soon).

**Comment by podsvirov** [comment]
Fixed in faf15a21ad760d2c05f428b4081c070bcd5f65f1.
